projects/PunktPunktStrich/index.md

```diff
@@ -0,0 +1,100 @@
+--- 
+authors: 
+  - "Malte Völkner"
+authors-url: 
+  - "https://github.com/MalteVoelkner"
+layout: project
+title: PunktPunktStrich / ..-
```

Comment:
This might be a problem when not in quotes.  Do 

    "PunktPunktStrich / ..-"